install.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+cd /vagrant
+
+gpg --keyserver hkp://keys.gnupg.net --recv-keys D39DC0E3
+curl -L https://get.rvm.io | bash -s stable
+
+source $HOME/.rvm/scripts/rvm
+rvm use --install 2.2
+
+bundle update
+bundle install
+
+# # set up the DB
+cp config/database.yml.example config/database.yml
+bundle exec rake db:setup
+
+bundle exec sidekiq &
+
+bundle exec rails s
```

Comment:
Instead of backgrounding `bundle exec sidekiq` and running `bundle exec rails s` in the foreground, it would be better to just run `foreman start`, which uses the commands in the `Procfile` and provides log output that is easier to decipher.

(Foreman isn't in the Gemfile, so it's worth adding in the `development` group)